src/c360_client/utils.py

```diff
@@ -0,0 +1,51 @@
+import os
+import boto3
+import discreetly
+import json
+
+
+def get_boto_client(name, **kwargs):
+    """
+    Get a boto client. If you are within Colab and you have logged in,
+    use the project's stored AWS credentials. Otherwise, create boto
+    client as per normal.
+    """
+    if is_notebook_mode():
+        aws_access_key_id, aws_secret_access_key = get_aws_credentials()
+        aws_creds = dict(
+            aws_access_key_id=aws_access_key_id,
+            aws_secret_access_key=aws_secret_access_key,
+            region_name="ap-southeast-1",
+        )
+        return boto3.client(name, **aws_creds, **kwargs)
+    else:
+        return boto3.client(name, **kwargs)
+
+
+def get_secret_session():
+    cfg = {
+        "default": {
+            "type": "gcp",
+            "datastore_project": "acme-c360",
+            "keyid": "projects/acme-c360-kms/locations/global/keyRings/dev/cryptoKeys/default"
```

Comment:
should these be dynamic?

Reply:
addressed in https://github.com/tra-sg/c360-python-client/pull/1/commits/8e0c6e75592f031e6787175ffce867b2dc9fd57b